lib/floe/container_runner/docker.rb

```diff
@@ -102,6 +103,7 @@ def status!(runner_context)
         return if runner_context.key?("Error")
 
         runner_context["container_state"] = inspect_container(runner_context["container_ref"])&.dig("State")
+        runner_context["container_state"]
```

Comment:
I don't understand the need for this line

Reply:
Same - the previous line returns that same value, so this is not needed.

Reply:
My mistake I had some debug code in here to dump the state and this was left over, great catch